config/urls.py

```diff
@@ -21,12 +22,11 @@
     url(r'^app/', include('seed.urls', namespace="seed", app_name="seed")),
 
     # root configuration items
-    url(r'^eula/', include('tos.urls', namespace='tos', app_name='tos')),
     url(r'^i18n/', include('django.conf.urls.i18n')),
     url(r'^robots\.txt', robots_txt, name='robots_txt'),
 
     # API
-    url(r'^api/swagger/', include('rest_framework_swagger.urls'), name='swagger'),
+    url(r'^api/swagger/', get_swagger_view(title='SEED API'), name='swagger'),
```

Comment:
A much nicer approach here.

Reply:
this was product of upgrading the swagger django rest framework.